Crash report 4.1: we've missed one fix for getString (int resId, Object... formatArgs)

We've missed to apply the fix around `getString (int resId, Object... formatArgs)` introduced in https://github.com/wordpress-mobile/WordPress-Android/pull/2760
in one place in Stats.

```
de.greenrobot.event.EventBusException: Invoking subscriber failed
       at de.greenrobot.event.EventBus.handleSubscriberException(EventBus.java:518)
       at de.greenrobot.event.EventBus.invokeSubscriber(EventBus.java:500)
       at de.greenrobot.event.EventBus.invokeSubscriber(EventBus.java:492)
       at de.greenrobot.event.HandlerPoster.handleMessage(HandlerPoster.java:67)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:194)
       at android.app.ActivityThread.main(ActivityThread.java:5392)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:525)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:833)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:600)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by: java.lang.AssertionError: couldn't initialize LocaleData for locale en-au
       at libcore.icu.LocaleData.initLocaleData(LocaleData.java:167)
       at libcore.icu.LocaleData.get(LocaleData.java:121)
       at java.util.Formatter.format(Formatter.java:1039)
       at java.util.Formatter.format(Formatter.java:1009)
       at java.lang.String.format(String.java:1988)
       at android.content.res.Resources.getString(Resources.java:350)
       at android.app.Fragment.getString(Fragment.java:797)
       at org.wordpress.android.ui.stats.StatsFollowersFragment.getTotalFollowersLabel(StatsFollowersFragment.java:242)
       at org.wordpress.android.ui.stats.StatsFollowersFragment.updateUI(StatsFollowersFragment.java:208)
       at org.wordpress.android.ui.stats.StatsAbstractListFragment.onEventMainThread(StatsAbstractListFragment.java:437)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:525)
       at de.greenrobot.event.EventBus.invokeSubscriber(EventBus.java:498)
       at de.greenrobot.event.EventBus.invokeSubscriber(EventBus.java:492)
       at de.greenrobot.event.HandlerPoster.handleMessage(HandlerPoster.java:67)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:194)
       at android.app.ActivityThread.main(ActivityThread.java:5392)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:525)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:833)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:600)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

--- a/WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java
@@ -238,7 +238,7 @@ public class StatsFollowersFragment extends StatsAbstractListFragment {
     }
 
     private String getTotalFollowersLabel(int total) {
-        String totalFollowersLabel;
+        final String totalFollowersLabel;
 
         if (mTopPagerSelectedButtonIndex == 0) {
             totalFollowersLabel = getString(R.string.stats_followers_total_wpcom);

--- a/WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java
@@ -238,11 +238,15 @@ public class StatsFollowersFragment extends StatsAbstractListFragment {
     }
 
     private String getTotalFollowersLabel(int total) {
-        if ( mTopPagerSelectedButtonIndex == 0 ) {
-            return getString(R.string.stats_followers_total_wpcom, FormatUtils.formatDecimal(total));
+        String totalFollowersLabel;
+
+        if (mTopPagerSelectedButtonIndex == 0) {
+            totalFollowersLabel = getString(R.string.stats_followers_total_wpcom);
+        } else {
+            totalFollowersLabel = getString(R.string.stats_followers_total_email);
         }
 
-        return  getString(R.string.stats_followers_total_email, FormatUtils.formatDecimal(total));
+        return String.format(totalFollowersLabel, FormatUtils.formatDecimal(total));
     }
 
     @Override

